mobile_verifier/src/reward_shares.rs

```diff
@@ -601,7 +789,7 @@ mod test {
             total += *val
         }
 
-        assert_eq!(total, 416_666_666_666_669); // total emissions for 1 hour
+        assert_eq!(total, 4_166_666_666_662); // total emissions for 1 hour
     }
 
```

Comment:
Can you add a test to cover the dc_to_mobile_bones, given that is a key function we want to ensure any potential future behaviour changes is covered

Reply:
I added a test for bytes -> bones, which goes through `dc_to_mobile_bones`, so it should test both